docs/reference/twig_helpers.rst

```diff
@@ -15,6 +15,11 @@ Render a page url
     {{ url(page) }} => https://sonata-project.org/absolute/url/to/url
 
     {{ url(page, {}, true) }} => //sonata-project.org/network/path/to/url
+    
+.. note::
+
+    You can use ``Technical Alias`` to link with the pages e.g:
```

Comment:
Can you please explain what a *Technical Alias* is? AFAIK we do not mention that term in our docs.

Reply:
Hey @core23 

in sonata page, there is a filed called `Technical Alias`

https://github.com/sonata-project/SonataPageBundle/blob/7330ed5548d3b240df78e4dc2bc0f65e4cfa1f11/src/Admin/PageAdmin.php#L341

But It wasn't clear that I could use as router in twig `path`.

and I putted `Technical Alias` because it's showing me this way

<img width="759" alt="Screenshot 2021-04-26 at 20 43 41" src="https://user-images.githubusercontent.com/6358755/116134842-a8c7b200-a6d0-11eb-9041-3a6101828c20.png">

Reply:
I know that there's a field called **Technical Alias**. I'm not sure if this is clear to everyone who reads the docs